### PR TITLE
Initial recipes

### PR DESCRIPTION
**Pull request by ramereth**
This depends on osuosl-cookbooks/osl-letsencrypt-boulder-server#1 being merged.

### CODE REVIEW

**Comment by ramereth** [comment]
[osl-acme wrapper cookbook](https://trello.com/c/voRrd0Pn/443-osl-acme-wrapper-cookbook)

File: Berksfile [hunk 1]
```diff
@@ -1,3 +1,7 @@
 source 'https://supermarket.chef.io'
 
+cookbook 'osl-letsencrypt-boulder-server',
+         git: 'git@github.com:osuosl-cookbooks/osl-letsencrypt-boulder-server.git',
+         branch: 'ramereth/cleanup'
```
Comment (doublej472) [inline Berksfile]: Once this branch is merged wont this need to be removed?

Comment (ramereth) [inline Berksfile]: Correct.

**Comment by ramereth** [comment]
!bump major

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v1.0.0.  Have a nice day!

**Pull request merged**